README.rst

```diff
@@ -1095,6 +1096,16 @@ Choose from the list of available rules:
   Each namespace use MUST go on its own line and there MUST be one blank
   line after the use statements block.
 
+* **single_line_comment_style**
+
+  Converts multi-line comments that have only one line of actual content
+  into single-line comments, and hash comments to slash.
+
+  Configuration options:
+
+  - ``comment_type`` (``'all'``, ``'hash'``, ``'star'``): fix star comment ``/* */`` [``star``],
```

Comment:
In another PR @julienfalque [asked me](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2857#discussion_r122779498) to switch from booleans to one option with multiple values (for a valid reason) and I supposed this to be the preferred way.
I have no preference, if you ask for it again I'll change